File: SunnyWeather-pro/src/main/java/com/example/sunnyweather_pro/logic/model/PlaceResponse.java
package com.example.sunnyweather_pro.logic.model;

import java.util.List;

/*城市地址响应数据*/
public class PlaceResponse {
    //响应状态
    public String status;
    //地址数据包
    public List<Place> places;

    public PlaceResponse(String status, List<Place> places) {
        this.status = status;
        this.places = places;
    }
}

File: SunnyWeather-kotlin/src/main/java/com/driverskr/sunnyweather_kotlin/logic/model/PlaceResponse.kt
package com.driverskr.sunnyweather_kotlin.logic.model

import com.google.gson.annotations.SerializedName

/**
 * @Author: driverSkr
 * @Time: 2023/11/6 15:06
 * @Description: 地址相关数据$
 */
data class PlaceResponse(val status: String, val places: List<Place>)

data class Place(val name: String, val location: Location,
                @SerializedName("formatted_address") val address: String)

data class Location(val lng: String, val lat: String)
